controllers/frontend_controller_suite_test.go

```diff
@@ -264,6 +268,8 @@ var _ = Describe("Frontend controller with service", func() {
 			}, timeout, interval).Should(BeTrue())
 			Expect(createdIngress.Name).Should(Equal(FrontendName))
 			Expect(createdIngress.Spec.Rules[0].HTTP.Paths[0].Backend.Service.Name).Should(Equal(ServiceName))
+			Expect(createdIngress.Annotations["nginx.ingress.kubernetes.io/whitelist-source-range"]).Should(Equal("192.168.0.0/24,10.10.0.0/24"))
```

Comment:
I hate that one of these is comma-separated and the other is space-separated. Why must they do this to us?